dockstore-webservice/src/test/resources/multi-version-cwl/many-version.cwl

```diff
@@ -0,0 +1,59 @@
+cwlVersion: v1.0.dev4
```

Comment:
this version looks malformed

Reply:
Indeed, it does look malformed, but it's a valid CWL version, per this list: https://www.commonwl.org/v1.2/CommandLineTool.html#CWLVersion

Reply:
gross